compiler/parser/src/lexer.rs

```diff
@@ -683,9 +681,24 @@ where
             } else {
                 StringKind::Normal
             };
+
+            let prefix = if is_raw && is_fstring {
+                Some("rf".to_string())
+            } else if is_raw {
+                Some('r'.to_string())
+            } else if is_fstring {
+                Some('f'.to_string())
+            } else if is_unicode {
+                Some('u'.to_string())
+            } else {
+                None
+            };
```

Comment:
Ref: https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals

Reply:
Would you put the reference to the source code comments?

Reply:
@youknowone Added!